ophyd/ophydobj.py

```diff
@@ -54,8 +63,8 @@ class OphydObject:
 
     _default_sub = None
 
-    def __init__(self, *, name=None, parent=None, labels=None,
-                 kind=None):
+    def __init__(self, *, name = None, parent = None, labels = None,
```

Comment:
The original was correct, PEP8 wise. Assignment statements get spaces, but keyword arguments do not. Example:

```py
def f(x=3):
    y = x + 1
```

Reply:
went back to the original